src/schema/v2/partner.ts

```diff
@@ -482,6 +484,50 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLString,
         resolve: ({ default_profile_id }) => default_profile_id,
       },
+      documentsConnection: {
+        description: "A connection of documents at a partner.",
+        type: partnerDocumentsConnection.connectionType,
+        args: pageable({
+          partnerID: {
```

Comment:
**question**: If we're nesting within the partner type, I don't think we'll need to support this explicit partnerID query field argument. Should be able to grab the partner ID from the `_root` parameter in the resolve function.